README.md

```diff
@@ -1 +1,221 @@
-# rust-wasm-template
\ No newline at end of file
+<div align="center">
+  <a href="https://github.com/fission-codes/rust-template" target="_blank">
+    <img src="./assets/logo.png" alt="rust-template Logo" width="100"></img>
+  </a>
+
+  <h1 align="center">rust-template</h1>
+
+  <p>
+    <a href="https://github.com/fission-codes/rust-template/actions?query=">
+      <img src="https://github.com/fission-codes/rust-template/actions/workflows/build.yml/badge.svg" alt="Build Status">
+    </a>
+    <a href="./LICENSE-APACHE">
+      <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="License-Apache">
+    </a>
+    <a href="./LICENSE-MIT">
+      <img src="https://img.shields.io/badge/License-MIT-blue.svg" alt="License-MIT">
+    </a>
+    <a href="https://discord.gg/AunfpqKUHU">
+      <img src="https://img.shields.io/static/v1?label=Discord&message=join%20us!&color=mediumslateblue" alt="Discord">
+    </a>
+  </p>
+</div>
+
+<div align="center"><sub>:warning: Work in progress :warning:</sub></div>
+
+##
+
+This is Fission's opinionated [Rust][rust] and [Rust][rust]+[WebAssembly][wasm]
```

Comment:
> This __ is Fission's opinionated

This sentence could use a noun where at the `__`. Maybe "This template" or "This repository"?